queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM vet_clinic WHERE name LIKE '%mon';
+SELECT name FROM vet_clinic WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT name FROM vet_clinic WHERE neutured = true AND escape_attempts < 3;
+SELECT date_of_birth FROM vet_clinic WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM vet_clinic WHERE weight_kg > 10.5;
+SELECT * FROM vet_clinic WHERE neutured = true;
+SELECT * FROM vet_clinic WHERE name != 'Devimon';
```

Comment:
- [ ] At this point, line 9 needs some assistance, you are asked to find animals that are not named `Gabumon` not Devimon.
<img width="1028" alt="1" src="https://github.com/Omwomo/vet_clinic_databases/assets/77843050/494fd9ab-3a30-4c6a-a08b-cf687b5eaa3f">